Lib/test/test_logging.py

```diff
@@ -3926,6 +3926,32 @@ def test_config_queue_handler(self):
             msg = str(ctx.exception)
             self.assertEqual(msg, "Unable to configure handler 'ah'")
 
+    @unittest.skipIf(support.is_wasi, "WASI does not have multiprocessing.")
```

Comment:
Instead of checking specifically for WASI, tests that use subprocesses should use `@support.requires_subprocess()`.

Reply:
But this test doesn't use `subprocess`, only `multiprocessing` - I'm not sure if that `requires_subprocess` is appropriate. I can't (yet) tell whether the problem is inherent to the platform (iOS ARM64 Simulator) or, if I need to tighten the skip conditions for the test, exactly what the condition should be.

Reply:
The error on iOS is exactly the same: see the bottom of the traceback above. The fact that `multiprocessing` isn't available on iOS is already [mentioned in the documentation](https://docs.python.org/3.13/library/multiprocessing.html).

An even more specific check would be `support.skip_if_broken_multiprocessing_synchronize()`, which is already used a few times in test_logging.

Reply:
> The error on iOS is exactly the same

Yes, I missed that for some reason. Perhaps the answer is to add a `support.skip_if_no_multiprocessing`, as `support.skip_if_broken_multiprocessing_synchronize` seems too specific for this particular test.

Reply:
Yes, I've already done that :smile: